elasticache.tf

```diff
@@ -1,7 +1,7 @@
 
 resource "aws_elasticache_cluster" "noMemcachedInElastiCache" {
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure ElastiCache for Memcached is not in use in AWS PCI DSS environments**
 
**Vulnerability**
ElastiCache for Memcached is not in use in AWS PCI DSS environments
 
**Remediation**
The AWS PCI DSS Compliance program only includes Amazon ElastiCache for Redis as a PCI-compliant service. Hence, change engine to Redis

source: accurics-onprem